tests/fgnhg_test.py

```diff
@@ -9,187 +9,45 @@
-| FG_NHG_PREFIX   | Next Hop           | Hash buckets   |
-+=================+====================+================+
-| fc:05::/128     | 200:200:200:200::4 | 0              |
-|                 |                    | 1              |
-|                 |                    | 2              |
-|                 |                    | 3              |
-|                 |                    | 4              |
-|                 |                    | 5              |
-|                 |                    | 6              |
-|                 |                    | 7              |
-|                 |                    | 8              |
-|                 |                    | 9              |
-|                 |                    | 10             |
-|                 |                    | 11             |
-|                 |                    | 12             |
-|                 |                    | 13             |
-|                 |                    | 14             |
-|                 |                    | 15             |
-+-----------------+--------------------+----------------+
-| fc:05::/128     | 200:200:200:200::5 | 16             |
-|                 |                    | 17             |
-|                 |                    | 18             |
-|                 |                    | 19             |
-|                 |                    | 20             |
-|                 |                    | 21             |
-|                 |                    | 22             |
-|                 |                    | 23             |
-|                 |                    | 24             |
-|                 |                    | 25             |
-|                 |                    | 26             |
-|                 |                    | 27             |
-|                 |                    | 28             |
-|                 |                    | 29             |
-|                 |                    | 30             |
-|                 |                    | 31             |
-+-----------------+--------------------+----------------+
+Alias     Next Hop            Hash buckets
+--------  ------------------  ----------------------------
+fgnhg_v6  200:200:200:200::4  0, 1, 2, 3, 4, 5, 6, 7
+fgnhg_v6  200:200:200:200::5  8, 9, 10, 11, 12, 13, 14, 15
 """
 
 show_fgnhg_active_hops_output="""\
-+-----------------+--------------------+
-| FG_NHG_PREFIX   | Active Next Hops   |
-+=================+====================+
-| fc:5::/128      | 200:200:200:200::4 |
-|                 | 200:200:200:200::5 |
-+-----------------+--------------------+
-| 100.50.25.12/32 | 200.200.200.4      |
-|                 | 200.200.200.5      |
-+-----------------+--------------------+
+FG NHG Prefix    Active Next Hops
+---------------  --------------------------------------
+fc:5::/128       200:200:200:200::4, 200:200:200:200::5
+100.50.25.12/32  200.200.200.4, 200.200.200.5
```

Comment:
This may end up as a lot of output on a single line, maybe we can limit how many we print on one line or separate each next-hop with a new line(as it was originally).